modin/pandas/test/test_io.py

```diff
@@ -1707,3 +1707,19 @@ def test_from_arrow():
     pandas_df = create_test_pandas_dataframe()
     modin_df = from_arrow(pa.Table.from_pandas(pandas_df))
     df_equals(modin_df, pandas_df)
+
+
+@pytest.mark.parametrize(
+    "kwargs",
+    [
+        {"names": [0, 1, 2, 3, 4, 5]},
+        {"names": [0]},
+        {"names": None, "usecols": [0, 1, 2]},
+        {"names": [0, 1, 2], "usecols": [0, 1, 2]},
```

Comment:
I know it does not matter implementation-wise, but for good measure let's test an unordered list as well.

Reply:
Changed to use random ordering for usecols and naming.